doc: updates for Generic Workflow to make a more clear explanation after Haldis Sylte's review. (#283)
equinor/ecalc: docs/docs/about/modelling/workflow/generic_workflow.md

### Validation

Before running an eCalc™ model, checking whether the eCalc™ model is valid or not is an essential task. If a model is not valid, this means that input requirements set by the user are not being fulfilled, or that some consumers are giving unrealistic solutions. 

> Validity can be checked by consumers, and there are often specific reasons why certain consumers are invalid. Therefore, it requires engineers not only to understand the valication of operation points (history data) from consumers, but also to know the specific limits of compressor or pump charts. The charts of consumers should fit their operation points. For example:

Reviewer: ```suggestion
Validity can be checked by consumers, and there are often specific reasons why certain consumers are invalid. Therefore, it requires engineers not only to understand the validation of operation points (history data) from consumers, but also to know the specific limits of compressor or pump charts. The charts of consumers should fit their operation points. For example:
```

- **Compressors and Pumps**: It is common that either too high a head or rate value is specified. This means that the invalid point is outside the limits of the performance chart. To determine the issue, it is recommended that the operational points (Head, and actual flowrate) are plotted together with the chart.
- **Generator Set**: The most common issue here is that the amount of power required is higher than the maximum value in the utilised genset. 

> Before any chart adjust, it suggests that engineers should have a check first with those who provide the input requirements for their eCalc™ model to confirm the validities make sense. 

Reviewer: ```suggestion
Before any chart adjustments are made, engineers should first verify who provide the input that the adjustments make sense. 
```

Something like that?
